main.cpp

```diff
@@ -5,16 +5,21 @@
 #include <opencv2/opencv.hpp>
 #include <opencv2/imgcodecs.hpp>
 #include "src/LaneDetection/detect.hpp"
+#include "src/LaneKeeping/lanekeeping.hpp"
+
+#define FRAME_WIDTH 510
+#define FRAME_HEIGHT 280
```

Comment:
delete both. And initialize ld and lk objects with witdth = image.cols and height = image.rows

Reply:
Done ✅
```c++
detect laneDetection(image.cols, image.rows, camera);
laneKeeping laneKeeping(image.cols, image.rows, camera);
```